### PR TITLE
practice2.py , practice3.py

### PR DESCRIPTION
**Pull request by samarth-shah**
issue solving : #10 
solved practice2 and practice3

### CODE REVIEW

File: practice2.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def mysum(*num):
```
Comment (nimit2801) [inline practice2.py]: Can you explain the use case of `(*num)` here?

**Review by nimit2801 — CHANGES_REQUESTED** [review]
Hey @samarth-shah , we can take **user input** in practice2.py and practice3.py

**Review by nimit2801 — CHANGES_REQUESTED** [review]
Also if possible can you add the questions in the header this would solve the conflict for a user to find the question. I'll also add this to the main issue so that it can be taken care of.

**Review by nimit2801 — APPROVED** [review]
LGTM! 😄 
Thanks for you contribution ✅

File: practice2.py (hunk 1 above)
Comment (samarth-shah) [reply practice2.py]: > 
> 
> Can you explain the use case of `(*num)` here?

*num allows you to do is take in more arguments than the number of formal arguments. With *num, any number of extra arguments can be tacked on to your current formal parameters.

**Comment by samarth-shah** [comment]
> 
> 
> Hey @samarth-shah , we can take **user input** in practice2.py and practice3.py

Yes we can take

File: practice2.py (hunk 1 above)
Comment (nimit2801) [reply practice2.py]: Thanks for the update! ✅

**Pull request merged**